libtenzir/builtins/operators/summarize.cpp

```diff
@@ -788,6 +845,13 @@ class summarize_operator final : public crtp_operator<summarize_operator> {
     return "summarize";
   }
 
+  auto input_independent() const -> bool override {
+    if (config_.created_timeout or config_.update_timeout) {
+      return true;
+    }
+    return false;
+  }
```

Comment:
```suggestion
  auto input_independent() const -> bool override {
    return config_.created_timeout || config_.update_timeout;
  }
```

Just because I have nothing else to say (: